utils/githooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# A git hook to validate  and correct the copyright date in source files.
+
```

Comment:
```suggestion
if [ -n "${NO_PRE_COMMIT:=""}" ]; then
       exit 0
fi
```

This can allow the hook to be disabled, to avoid updating copyright on files that don't really need it (happens sometimes when merging/rebasing).